pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -64,9 +64,15 @@ func checkBundleCached() error {
 }
 
 func fixBundleCached() error {
+	// Should be remove after 1.19 release
```

Comment:
`removed`
If we remove this  for 1.19, this means we don't consider upgrades from 1.16 to >1.19 to be a supported upgrade path.

Reply:
@cfergeau yes from 1.16 => 1.19 will not be supported in this case.